app/controllers/admin/conference_controller.rb

```diff
@@ -174,8 +174,25 @@ def edit
       end
     end
 
+    def conference_info
+      # To display sponsors in the conference wide information page
+      @sponsors = @conference.sponsors
+      @current_event_schedules = @program.selected_schedule.event_schedules.current
```

Comment:
Side by side.
But it would make total sense to show the track in the event details!